python/prompt_bench/containers/vllm.py

```
@@ -0,0 +1,81 @@
+        VLLM_IMAGE,
+        "--model",
+        f"/models/{model}",
+        "--served-model-name",
+        model,
+        "--gpu-memory-utilization",
+        str(gpu_memory_utilization),
+        "--max-model-len",
+        "4096",
+    ]
+    logger.info("Starting vLLM container with model: %s", model)
+    result = subprocess.run(command, capture_output=True, text=True, check=False)
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)